document_meta.go

```diff
@@ -357,6 +357,21 @@ func fieldName(sf reflect.StructField) (string, bool) {
 	return snaker.CamelToSnake(sf.Name), false
 }
 
+func isEmbedded(sf reflect.StructField) bool {
+	// anonymous structs are always embedded
+	if sf.Anonymous {
+		return true
+	}
+	// search for embedded tag
+	tags := strings.Split(sf.Tag.Get("db"), ",")
+	for i, tag := range tags {
+		if tag == "embedded" && i > 0 {
```

Comment:
maybe instead of `i > 0`, better to loop by index starting 1? 🤔

Reply:
To not accidentally step out of bounds